src/java/us/kbase/mobu/initializer/test/DockerClientServerTester.java

```diff
@@ -166,9 +168,36 @@ protected File initPython(String moduleName) throws Exception {
         return moduleDir;
     }
     
+    public static void correctDockerfile(File moduleDir) throws Exception {
+        File buildXmlFile = new File("build.xml");
+        File sdkSubFolder = new File(moduleDir, "kb_sdk");
+        File sdkDistSubFolder = new File(sdkSubFolder, "dist");
+        FileUtils.copyFile(buildXmlFile, new File(sdkSubFolder, buildXmlFile.getName()));
```

Comment:
They do create all subfolders when you copy:
http://grepcode.com/file/repo1.maven.org/maven2/commons-io/commons-io/1.0/org/apache/commons/io/FileUtils.java#409